examples/raylib/raylib.c

```diff
@@ -75,13 +81,30 @@ void draw_objects(tmx_object_group *objgr)
                 rect.y = head->y;
                 rect.width = head->width;
                 rect.height = head->height;
-                DrawRectangleLines(rect.x, rect.y, rect.width, rect.height, color);
+                DrawRectangleLines(rect.x -player.x, rect.y -player.y, rect.width, rect.height, color);
             } else if (head->obj_type  == OT_POLYGON) {
-                draw_polygon(head->content.shape->points, head->x, head->y, head->content.shape->points_len);
+                draw_polygon(head->content.shape->points, head->x -player.x, head->y -player.y, head->content.shape->points_len, color);
             } else if (head->obj_type == OT_POLYLINE) {
-                draw_polyline(head->content.shape->points, head->x, head->y, head->content.shape->points_len);
+
+                /* You have to make Vector2* out of double** data points */
```

Comment:
Personally I've found using `draw_polygon\draw_polyline` (both with Raylib's `DrawLine` inside) to be more elegant choice because you don't have to do any transformations to get proper Vector2* argument for Raylib's functions.

So I left this piece of code for users to make choice on their own.